pandas/io/sql.py

```diff
@@ -661,10 +662,21 @@ def to_sql(
     schema : str, optional
         Name of SQL schema in database to write to (if database flavor
         supports this). If None, use default schema (default).
-    if_exists : {'fail', 'replace', 'append'}, default 'fail'
+    if_exists : {'fail', 'replace', 'append'},
+        default 'fail'.
         - fail: If table exists, do nothing.
         - replace: If table exists, drop it, recreate it, and insert data.
         - append: If table exists, insert data. Create if does not exist.
+    on_conflict : {None, 'do_nothing', 'do_update'}, optional
```

Comment:
Can you specify the default arg similar to other params? What does `on_conflict=None` actually do when there are conflicts